core/main/src/firebolt/handlers/hdmi_rpc.rs

```diff
@@ -0,0 +1,198 @@
+use crate::{
+    firebolt::rpc::RippleRPCProvider, service::apps::app_events::AppEvents,
+    state::platform_state::PlatformState, utils::rpc_utils::rpc_err,
+};
+use jsonrpsee::{
+    core::{async_trait, RpcResult},
+    proc_macros::rpc,
+    RpcModule,
+};
+use ripple_sdk::api::{
+    device::device_events::{
+        DeviceEvent, DeviceEventCallback, DeviceEventRequest, HDMI_CONNECTION_CHANGED,
+    },
+    firebolt::{
+        fb_general::{ListenRequest, ListenerResponse},
+        panel::fb_hdmi::{
+            GetAvailableInputsResponse, HdmiOperation, HdmiSelectOperationRequest,
+            HdmiSelectOperationResponse,
+        },
+    },
+    gateway::rpc_gateway_api::CallContext,
+};
+use ripple_sdk::{
+    api::device::device_events::AUTO_LOW_LATENCY_MODE_SIGNAL_CHANGED, log::error, serde_json,
+};
+use ripple_sdk::{
+    api::device::panel::device_hdmi::HdmiRequest,
+    extn::extn_client_message::{ExtnPayload, ExtnResponse},
+};
+
+#[rpc(server)]
+pub trait Hdmi {
```

Comment:
Maybe the module should be called HdmiInput seeing as that is the name in the FB spec?

Reply:
Renamed it.